### DISCUSSION TITLE
hsrb_command error

### DISCUSSION THREAD

**Discussion by yutaka-takaoka** [HumanSupportRobot/QandA#12] [Q&A] [answered]
Hi support,

I am having a trouble with hsrb_command during the clean install.
When I hit "hsrb_command upload_robot_config", I saw this.
```
Script start.
Getting robot version information...
Getting phase information from server...
version:HSRC-PHASE1/18.10 does not exist
```

Is this because of too old software version?
Should I just skip this process and boot with the latest image?

Any advice would be appreciated.

**Comment by hsr-admin** [comment, marked as answer]
The "hsrb_command upload_robot_config" command is not available because the version of software you are using is out of date.
This command is used to upload robot-specific data such as calibration data to the server.
If you are not sure if the data on the server is the latest, please back up the data in HSR (/etc/opt/tmc/robot/) locally, such as on a PC, just to be safe.

**Reply by yutaka-takaoka** [reply]
Thank you for your quick reply. I understood.
